backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/snapshots/ColdMergeSnapshotSingleDiskCommand.java

```diff
@@ -113,6 +113,14 @@ public boolean performNextOperation(int completedChildCount) {
         }
     }
 
+    private RemoveSnapshotSingleDiskStep getInitialMergeStepForImage() {
+        if (getImageInfoFromVdsm(getDestinationDiskImage()) == null) {
+            log.info("Image has been previously merged and deleted, proceeding to DB sync");
```

Comment:
maybe the same more generic message should be used as below?
`"Image does not exist, attempting to synchronize the database"`

Reply:
Right :)